### ISSUE TITLE
I'm currently using segments.dblclick but it doesn't seem to work at all, I'm on version v3.4.2

### ISSUE THREAD

**Issue by WL1994** [bbc/peaks.js#553]
I'm currently using segments.dblclick but it doesn't seem to work at all, I'm on version v3.4.2

**Comment by chrisn** [comment]
Thanks for reporting this. Using the demo pages, I see segments.dblclick events in Chrome but not in Firefox. Which browser are you using? And which version of the Konva dependency do you have?

**Comment by WL1994** [comment]
@chrisn Thank you for your reply.  I am currently using Google Chrome on macOS, and the Konva version is 9.3.16.

**Comment by WL1994** [comment]
@chrisn  I seem to have found the problem. The reason why `segments.dblclick` is not working is that I set `playheadWidth: 2`. I feel like when I double-click, I'm clicking on the line rather than the segment. If that’s the case, please help me verify.

**Comment by chrisn** [comment]
You're right, that's what I found too, but I'm not sure why it works in some browsers and not others. I also noticed that sometimes double-clicks don't cause a `dblclick` event and you have to double-click twice to generate an event.

**Comment by chrisn** [comment]
Please try 4.0.0-beta.0.
